src/templates/static/legal/payment_agents.jsx

```diff
@@ -2,185 +2,182 @@ import React from 'react';
-                    <li>{it.L('Where the Agent is dissolved, or deregistered from any records of a formal register whatsoever. Furthermore, where any procedure is initiated, seeking or intending or proposing the Agent\'s dissolution or deregistration from any records of a formal register. ')}</li>
-                    <li>{it.L('Any event capable to have a material adverse effect on the Agent\'s ability to perform any of its duties and obligations in accordance with this Agreement.')}</li>
+                    <li>{it.L('In case of the Agent\'s death, incapacity, or mental unsoundness;')}</li>
+                    <li>{it.L('Where the Agent becomes incapable to pay his debts as they fall due, or is bankrupt or insolvent, as defined under any and all applicable bankruptcy or insolvency law where the Agent is an individual;')}</li>
+                    <li>{it.L('Where the Agent acts in breach of any warranty, representation, or promise made under this agreement, and any information provided to the Company in connection with this agreement is, or becomes, untrue or misleading;')}</li>
+                    <li>{it.L('Where the Agent fails to comply with the Agent\'s obligations or fails to perform any of the Agent\'s duties or other provisions under this agreement and such failure continues for at least one business day after the receipt of the non-performance notice given by the Company')}</li>
+                    <li>{it.L('Where any proceedings are involuntarily initiated against the Agent by any third parties who seek or propose liquidation, reorganisation, restructuring, an arrangement or composition, or a freeze or moratorium in relation to the Agent or the Agent\'s debts in accordance with the law')}</li>
+                    <li>{it.L('Where any law-suit, action, or other legal or administrative proceedings in connection to this agreement are initiated for any execution, attachment or garnishment, or distress against the Agent or where an encumbrance takes possession of the entire or any part of the Agent\'s property, undertakings, or assets, whether tangible and intangible')}</li>
+                    <li>{it.L('Where the Agent is dissolved or deregistered from any records of a formal register whatsoever; furthermore, where any procedure is initiated that is seeking, intending, or proposing the Agent\'s dissolution or deregistration from any records of a formal register')}</li>
+                    <li>{it.L('Any event capable to have a material adverse effect on the Agent\'s ability to perform any of its duties and obligations in accordance with this agreement')}</li>
```

Comment:
@AzitaRassi In this list, the first 3 list items end with `;` and the rest don't. Is that correct?

Reply:
I'm not sure if I'm looking at the right list, but from what I see on lines 134-136, there's only one semicolon, and that's before the word "furthermore". There's no punctuation mark after the last words ("law", "intangible", and "register"). If I'm looking at the wrong items, please let me know :-)

Reply:
Oh I meant the list starting from line 130 to 137

Reply:
You're completely right. I made the changes. Thanks.

Reply:
Alright